java/ql/src/change-notes/2022-09-13-flow-model-for-uri-constructors.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added taint model for arguments of `java.net.URI` constructors.
```

Comment:
This change note is a bit imprecise - the new models are only for TaintedPath.ql

Reply:
I added some more detail.